setup.py

```diff
@@ -165,7 +165,7 @@ def run(self):
         'sympy',
         'numpy',
         'cython',
-        'yt>=3.6.0',
+        'yt>=4.0.0',
```

Comment:
since this requirement can not be satisfied atm, I suggest to make this a draft PR

Reply:
I'd recommend just reverting this change so the testing can run. I'd like to try and merge this PR sooner rather than later.